aws_lambda_builders/actions.py

```diff
@@ -117,7 +123,6 @@ def __init__(self, source_dir, artifact_dir, destination_dir, excludes=None):
         self.source_dir = source_dir
         self.artifact_dir = artifact_dir
         self.dest_dir = destination_dir
-        self.excludes = excludes or []
```

Comment:
Why do we still have this in constructor if we are removing this?

Reply:
Sound good, removed.